logger_test.go

```diff
@@ -0,0 +1,36 @@
+package analytics
+
+import (
+	"bytes"
+	"errors"
+	"log"
+	"testing"
+)
+
+// This test ensures that the interface doesn't get changed and stays compatible
```

Comment:
Not sure this needs to be in a test since there's no point running this (it breaks at compile time).